automation/gce-deploy/docker-compose.yml

```diff
@@ -28,6 +28,17 @@ services:
     links:
       - sqlproxy:sqlproxy
 
+  userinterface:
+      image: gcr.io/TEMPLATE_VAR_DOCKER_PROJECT/leonardo-ui:TEMPLATE_VAR_DOCKER_TAG
```

Comment:
Will this be substituted? I thought you need some kind of `$` syntax for docker-compose env substitutino.